Sources/Views/Components/SignInView.swift

```diff
@@ -0,0 +1,34 @@
+import GDSCommon
+import UIKit
+
+class SignInView: NibView {
+
+    var viewModel: SignInViewModel
```

Comment:
`final class` and `let viewModel`?

Reply:
view model needs to be a var as its contents are modified elsewhere.  changed to final